src/middleware/auth.ts

```diff
@@ -0,0 +1,32 @@
+//const jwt = require("jsonwebtoken");
+
+import { NextFunction, Request, Response } from "express";
+import jwt from "jsonwebtoken";
+
+interface Verified {
```

Comment:
I would change the name here to `VerifiedToken` or something else more descriptive